src/manifest.json

```diff
@@ -17,7 +17,8 @@
 
   "background": {
     "service_worker": "background.ts",
-    "type": "module"
+    "type": "module",
+    "scripts": ["background.ts"]
```

Comment:
In Manifest v3 extensions, Firefox doesn't support background service workers, and Chrome doesn't support background scripts, so we need to specify both 🤦